pkgs/development/tools/misc/itstool/default.nix

```diff
@@ -10,6 +10,22 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ (python3.withPackages(ps: with ps; [ libxml2 ])) ];
 
+  # bin/itstool's shebang is "#!${python3.withPackages(...)/bin/python} -s"
+  # withPackages' shebang is "#!#{bash}/bin/bash -e
+  #
+  # macOS won't allow the target of a shebang to be an interpreted script,
+  # causing bin/itstool to get interpreted as bash.
+  #
+  # By prefixing /usr/bin/env to the shebang, we have env fork/exec the python
+  # wrapper, which is perfectly happy to execute an interpreted script.
+  #
+  # However, we don't want to do this on Linux, which only allows one argument
+  # in a shebang.
```

Comment:
This sounds like something we can update in patch-shebangs.sh. I didn't realize that /usr/bin/env worked differently! For instance, skipping it here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/patch-shebangs.sh#L64